components/collection/drop/HolderOfGenerative.vue

```diff
@@ -137,24 +77,27 @@ const { toast } = useToast()
 const { accountId, isLogIn } = useAuth()
 
 const { client } = usePrefix()
-const selectedImage = ref<string>('')
 const isLoading = ref(false)
 const isImageFetching = ref(false)
 const isAddFundModalActive = ref(false)
 
+const {
+  defaultName,
+  defaultImage,
+  defaultMax,
+  collectionId,
+  chainName,
+  disabledByBackend,
+  token,
+} = useGenerativeDropDetails(props.drop)
```

Comment:
`Argument of type 'Record<string, any>' is not assignable to parameter of type 'DropItem'.ts(2345)`

Reply:
fixed